README.rst

```diff
@@ -94,6 +94,12 @@ Version 1.4.1
 
 To be released.
 
+- Python 3.7 can be supported by updating flake8-import-order to 0.18.
+  See its changelog_ to know more about
+  updates.
+
+.. _changelog: https://github.com/PyCQA/flake8-import-order/blob/master/CHANGELOG.rst#018-2018-07-08
```

Comment:
The label is too general; better to use an [anonymous link target][1].

[1]: http://docutils.sourceforge.net/docs/user/rst/quickref.html#indirect-hyperlink-targets